addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionScanRuleUnitTest.java

```diff
@@ -49,9 +49,9 @@ protected int getRecommendMaxNumberMessagesPerParam(AttackStrength strength) {
             default:
                 return recommendMax + 14;
             case HIGH:
-                return recommendMax + 22;
+                return recommendMax + 24;
             case INSANE:
-                return recommendMax + 5;
+                return recommendMax + 7;
```

Comment:
I guess the Insane one doesn't really matter.
I hope/assume the High one isn't a huge deal since we were already over budget on this rule and still remain within it on Medium (which is generally default).

Reply:
I'm good with that :)